<!-- -->
<launch>
  <arg name="earth_frame_id" default="ground_ENU"/>
  <arg name="baselink_translation_frame_id" default="base_link_no_orientation"/>

  <arg name="publish_tf" default="false"/>
  <arg name="global_rtk_position_topic" default="global_rtk_position"/>
  <arg name="local_rtk_enu_position_topic" default="local_rtk_enu_position"/>

  <arg name="rtk_reference_geopoint_topic" default="rtk_reference_geopoint"/>


  <node name="gps_to_local_euclidean" pkg="gps_to_local_euclidean" type="gps_to_local_euclidean.py" respawn="true" output="screen">
    <param name="publish_tf" type="bool" value="$(arg publish_tf)" />
    <param name="earth_frame_id" type="str" value="$(arg earth_frame_id)" />
    <param name="baselink_translation_frame_id" type="str" value="$(arg baselink_translation_frame_id)" />
    <remap from="global_rtk_position" to="$(arg global_rtk_position_topic)"/>
    <remap from="local_rtk_enu_position" to="$(arg local_rtk_enu_position_topic)"/>
    <remap from="rtk_reference_geopoint" to="$(arg rtk_reference_geopoint_topic)"/>
  </node>
</launch>
